app/presenters/content_item/contents_list.rb

```diff
@@ -22,5 +28,19 @@ def extract_headings_with_ids(html)
       end
       headings.compact
     end
+
+    def first_item_has_long_content?
+      first_item_content.count > WORD_COUNT_LIMIT
+    end
+
+    def first_item_content
+      element = Nokogiri::HTML(body).css('p').first
```

Comment:
Some questions about this part:

* what happens if the content begins with something that's not a paragraph, eg a table or image?
* is this relatively slow at the moment? how performant could we make it?

Reply:
Good point about the paragraphs - I'll try and find some Detailed Guides examples with other content and work something out.
Steve's got a suggestion for optimising the Nokogiri stuff below 👇